src/__init__.py

```diff
@@ -142,8 +148,6 @@ def unhandled_exception(exc):
         tf_log.exception("Unhandled Exception: %s", (exc))
         return "Unhandled Exception: {}\n".format(exc), 500
 
-    tf_app.register_blueprint(views)
```

Comment:
I’m wondering why this line was removed

Reply:
Good catch, thanks! It was unintentional due to git madness when I was rearranging some things